src/routes/safe/components/Transactions/TxList/TxInfoCreation.tsx

```diff
@@ -21,32 +27,9 @@ export const TxInfoCreation = ({ transaction }: { transaction: Transaction }): R
   return (
     <TxDetailsContainer>
       <div className="tx-summary">
-        <div className="tx-hash">
-          <Text size="xl" strong as="span">
-            Transaction hash:{' '}
-          </Text>
-          <InlineEthHashInfo
-            textSize="xl"
-            hash={txInfo.transactionHash}
-            shortenHash={8}
-            showCopyBtn
-            explorerUrl={getExplorerInfo(txInfo.transactionHash)}
-          />
-        </div>
-        <div className="tx-created">
-          <Text size="xl" strong as="span">
-            Created:{' '}
-          </Text>
-          <Text size="xl" as="span">
-            {formatDateTime(timestamp)}
-          </Text>
-        </div>
-      </div>
-      <div className="tx-details">
-        <div className="tx-creator">
-          <Text size="xl" strong>
-            Creator:{' '}
-          </Text>
+        <TxDataRow title="Transaction hash:" value={txInfo.transactionHash} inlineType="hash" />
+        <TxDataRow title="Created:" value={formatDateTime(timestamp)} />
+        <TxDataRow title="Creator:" value={null}>
```

Comment:
What's with the `null` value? I think this shows why it being optional makes sense.